doc/user_guide/customization.rst

```diff
@@ -566,3 +566,145 @@ it is rendererd, you can set ``width`` or ``height`` to the string ``"container"
 Note that this will only scale with the container if its parent element has a size determined
 outside the chart itself; For example, the container may be a ``<div>`` element that has style
 ``width: 100%; height: 300px``. 
+
```

Comment:
This page is so nice to read with the collapsed tables! I noticed there were just a few sections that were missing a sentence introducing the tables, could we add these:

Can we use the same wording for projection and selection as for scale ranges ("Projections can be configured using..."):

![image](https://user-images.githubusercontent.com/4560057/208460217-2de7daa6-2801-472e-b672-f0f2740d91d3.png)

The table in the Header section is missing a leading sentence, could we add something like "Additional properties are summarized in the following table:"

![image](https://user-images.githubusercontent.com/4560057/208460719-04d011ad-43f0-4f33-b855-31eb783dfa2f.png)